domFoundations/events,storage,attributes/trigBox.js

```diff
@@ -0,0 +1,17 @@
+import { saveData } from "./storage.js";
+import { loadData } from "./storage.js";
+
+const trigBox = {
+    trigger: 'trigger' in localStorage ? loadData('trigger'): 'on',
+    trigSwitch() {
+        this.trigger = this.trigger === 'on' ? '' : 'on';
+        console.log(`Trigger value: ${this.trigger}`);
+        saveData('trigger', this.trigger);
+    },
+    trigStatusCheck() {
+        return this.trigger;
+    }
+}
+
+export const trigCheck = trigBox.trigStatusCheck.bind(trigBox)
```

Comment:
do not use .bind

Reply:
I used bind to preserve the method call context. Otherwise it would not work when I pass this method as an argument. I thought it is a good practice to use `.bind` for this purpose. If this is not the case, please advise what should I use instead?

By the way, you often suggest not to use `.bind` - why is it bad to use it, or do I use it wrong?